CRM/Utils/System/Base.php

```diff
@@ -1132,4 +1132,21 @@ public function getContactDetailsFromUser($uf_match):array {
     return $contactParameters;
   }
 
+  /*
```

Comment:
style issue:
` 1143 | ERROR | [x] You must use "/**" style comments for a function comment`

Reply:
Fixed. I'd try to skip running all the tests again if I knew how. Just for one asterix.

Reply:
One star but 4 stars for the good code cleanup efforts.